draft-bonaventure-quic-atsss-overview.mkd

```diff
@@ -45,11 +45,6 @@ author:
   organization: Tencent America
   email: spencerdawkins.ietf@gmail.com
  -
-  ins: M. Kuehlewind
```

Comment:
You may want revert this change.

Reply:
Yes, I saw this and did :-)

Reply:
Actually I was sure if I should list myself as author as we already have way too many authors. I think it would be more appropriate to list some people as contributor in a separate section at the end.

Reply:
We can keep that list, but add a tag for editors as suggested in #81. 

I will make a PR with a proposal.